content/reference/gql-types.md

```diff
@@ -0,0 +1,122 @@
+All objects in the database have relationships to other objects.
+
+## Scalar filters
+
+Every field that has a basic type can be used as a filter.
+The filters that are available depend on the data type, but the behavior of the `String` filters corresponds closely to `DateTime` and `Int` filters.
+
+### String filters
+
+String properties have the following filters:
+
+| Filter                         | Description                                               | Example argument                                 |
+|--------------------------------|-----------------------------------------------------------|--------------------------------------------------|
+| `eq`                           | Equals (exact match)                                      | `(filter: {id: {eq: "match"}})`                  |
+| `in`                           | From this match of arrays                                 | `(filter: {id: {in: ["dough", "cookie_unit"]}})` |
+| `lt`,`gt`,`le`, `ge` `between` | Less than, greater than, or between a lexicographic range | `(filter: {id: {lt: "M"}`                        |
+| `regexp`                       | A regular expression match using [`RE2` syntax](https://github.com/google/re2/wiki/Syntax/)                | `(filter: {id: {regexp: "/hello/i"}})`           |
+| `anyoftext`                    | A match for any entered strings, separated by spaces      | `(filter: {id: {anyoftext: "100 MD"}})`          |
+| `alloftext`                    | A match for all entered strings                           | `(filter: {id {alloftext: "MD"}})`               |
+
+### Integers, floats, DateTimes
+
+Properties that have a type of `Int`, `Float`, or `DateTime` can be filtered by the following keywords.
+Note that all date times are in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format.
+
+ - `lt`
+ - `le`
+ - `eq`
+ - `in`
+ - `between`
+ - `ge`
+ - `gt`
+
+Each keyword has the same behavior as described in [string filters](#string-filters), only they operate on numerical rather than lexicographic values.
+
+### Enum filters
+
+Properties of the type `Enum` can be filtered by the following:
+ - `lt`
+ - `le`
+ - `eq`
+ - `in`
+ - `between`
+ - `ge`
+ - `gt`
+
+Each keyword has the same behavior as described in [string filters](#string-filters).
+
+### Boolean filters
+
+Boolean filters can be either `true` or `false`.
+
+### Geolocation filters
+
+Geolocation filters return objects within specified geographic coordinates.
+They return matches within the specified [GeoJSON polygon](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6).
+
+| Filter     | Description                                      |
+|------------|--------------------------------------------------|
+| near       | Within the specified `distance` from the polygon |
+| within     | In the polygon coordinates                       |
+| Intersects | In the intersection of two polygons              |
```

Comment:
There is also Contains here.

I'd note that at the moment we don't seem to have any `Point` or `Polygon` (the types that implement this) that have `@search` defined.